src/index.ts

```diff
@@ -5,12 +5,24 @@ import QGISParser from 'geostyler-qgis-parser';
 // import OpenLayersParser from "geostyler-openlayers-parser";
 import MapfileParser from 'geostyler-mapfile-parser';
 import MapboxParser from 'geostyler-mapbox-parser';
-import { promises as fs } from 'fs';
+import {
+  existsSync,
+  lstatSync,
+  promises,
+  readdir
+} from 'fs';
 import minimist from 'minimist';
 import { StyleParser } from 'geostyler-style';
-import ora from 'ora';
+import ora, { Ora } from 'ora';
 import { logHelp, logVersion } from './logHelper';
 
+const ensureTrailingSlash = (inputString: string): string => {
+  if (!inputString) {
+    return inputString;
```

Comment:
This way the returntype of the function is depending on the input. I'd prefer to return an empty string here.

```suggestion
    return '';
```